custom_components/tuya_local/devices/diivoo_dwv010.yaml

```diff
@@ -25,22 +25,46 @@ secondary_entities:
         type: string
         name: operation
   - entity: sensor
-    name: Water use
     class: volume
+    name: Cumulative water usage
     category: diagnostic
     dps:
       - id: 112
         type: integer
         name: sensor
         unit: "l"
+  - entity: sensor
+    class: volume
+    name: Daily water usage
+    category: diagnostic
+    dps:
+      - id: 6
+        type: integer
+        name: sensor
+        unit: "l"
   - entity: sensor
     class: battery
+    name: Battery
     category: diagnostic
     dps:
       - id: 7
         type: integer
         name: sensor
         unit: "%"
+  - entity: sensor
```

Comment:
It isn't obvious what information these two are providing. I suspect it is not really useful enough to justify sensors, as free text cannot be tracked in any meaningful way, only numeric values and fixed sets of discrete values.

This is why I originally changed them to attributes, and I see no reason to change that.